### PR TITLE
add genai guide

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Add new GenAI use-case guide migrated from Google Doc.
https://deploy-preview-19262--docsdocker.netlify.app/guides/use-case/genai-video-bot/

### Related issues (optional)

Supersedes https://github.com/docker/docs/pull/19123
ENGDOCS-1942

### CODE REVIEW

**Comment by Davidnet** [comment]
LGTM!

File: content/guides/use-case/genai-video-bot/_index.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+   In the logs, you'll see the services are exposed on ports `8503` and `8504`.
+   The two services are complimentary to each other.
+
+   The `yt-whisper` service is running on port `8503`. This service feeds the
+   Pinecone database with videos that you want to archive in your knowledge
+   database. The following section explores this service.
+
+## Using the yt-whisper service
+
+The yt-whisper service is YouTube video processing service that uses the OpenAI
```
Comment (stephaurelio) [inline content/guides/use-case/genai-video-bot/_index.md:92]: Missing article:
```suggestion
The yt-whisper service is a YouTube video processing service that uses the OpenAI
```

File: content/guides/use-case/genai-video-bot/_index.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+Whisper model to generate transcriptions of videos and stores them in a Pinecone
+database. The following steps show how to use the service.
+
+1. Open a browser and access the yt-whisper service at [http://localhost:8503](http://localhost:8503).
+2. Once the application appears, in **Youtube URL** specify a Youtube video URL
```
Comment (stephaurelio) [inline content/guides/use-case/genai-video-bot/_index.md:97]: **Nit:** can maybe be clearer here
```suggestion
2. Once the application appears, in the **Youtube URL** field specify a Youtube video URL
```

File: content/guides/use-case/genai-video-bot/_index.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+2. In the **What do you want to know about your videos?** text box, ask the
+   Dockerbot a question about a video that was processed by the yt-whisper
+   service. The following example asks the question, "What is a sugar cookie?".
+   The answer to that question exists in the video processed in the previous
+   example,
+   [https://www.youtube.com/watch?v=yaQZFhrW0fU](https://www.youtube.com/watch?v=yaQZFhrW0fU).
+   
+   ![Asking a question to the Dockerbot](images/bot.webp)
+   
+   As you can see in the previous image, the Dockerbot answers the question and
```
Comment (stephaurelio) [inline content/guides/use-case/genai-video-bot/_index.md:141]: **Thought:** I wonder if the wording for this should be updated as some users might not view the image and may be using a screenreader, so they'll be relying on this paragraph to explain the example: 
```suggestion
   In this example, the Dockerbot answers the question and
```

**Review by stephaurelio — APPROVED** [review]
Super cool! I like how fast-paced it is with the option to learn more about the technologies at the end. Left a couple tiny comments.

**Review by stephaurelio — APPROVED** [review]
LGTM!

**Pull request merged**